src/AppBundle/Event/EventInvitationHandler.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace AppBundle\Event;
+
+use AppBundle\Entity\Event;
+use AppBundle\Entity\EventInvite;
+use AppBundle\Mailjet\MailjetService;
+use AppBundle\Mailjet\Message\EventInvitationMessage;
+use Doctrine\Common\Persistence\ObjectManager;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+
+class EventInvitationHandler
+{
+    private $manager;
+    private $mailjet;
+    private $urlGenerator;
+
+    public function __construct(ObjectManager $manager, MailjetService $mailjet, UrlGeneratorInterface $urlGenerator)
+    {
+        $this->manager = $manager;
+        $this->mailjet = $mailjet;
+        $this->urlGenerator = $urlGenerator;
+    }
+
+    public function handle(EventInvitation $invitation, ?string $ip, Event $event)
+    {
+        foreach ($invitation->guests as $guest) {
+            $invite = EventInvite::create($invitation->firstName, $invitation->lastName, $guest, $ip, $event);
+            $url = str_replace('localhost', 'en-marche.fr', $this->urlGenerator->generate('app_committee_show_event', [
```

Comment:
We should not change the context as other urls should not be changed, this one is meant to be sent by email, not printed in any page.